libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -27,6 +27,6 @@ export class FunctionClient implements IFunctionClient {
     if (!response.Payload) {
       return "";
     }
-    return toUtf8(response.Payload);
+    return JSON.parse(toUtf8(response.Payload));
```

Comment:
If we just add this line (without changing the signature of `invoke` in `cloud/function.ts`) does the fix still work?

Reply:
To be true, I made this change to give the possibility to return any type, not only string

As the documentation of cloud.Function it's declare as any

Reply:
To be honest, I made this change after reviewing the documentation (where the invoke function returns a JSON), and this modification allows for the return of any type of value.